core/store/src/trie/mod.rs

```diff
@@ -623,7 +678,10 @@ impl WrappedTrieChanges {
     ) -> Result<(), Box<dyn std::error::Error>> {
         self.insertions_into(&mut store_update)?;
         self.state_changes_into(&mut store_update)?;
-        store_update.set_ser(ColTrieChanges, self.block_hash.as_ref(), &self.trie_changes)?;
+        // TODO #2564
```

Comment:
are you decided not to apply https://github.com/nearprotocol/nearcore/pull/2592 to this change?

Reply:
correct